### DISCUSSION TITLE
Update Usage/text/#unicode

### DISCUSSION THREAD

**Discussion by jidanni** [ImageMagick/ImageMagick#4197] [General]
http://www.imagemagick.org/Usage/text/#unicode says
>if you can type unicode characters into commands or scripts you can use them directly..
>...é è à ù ç

But I can type 中文 directly, but the example fails for that.
>However few people [**do**] have keyboards or editors properly set up to handle unicode character input.

But now it is 2020, and few people **don't**!

**Comment by urban-warrior** [comment]
Did you specify a proper Unicode font?  Otherwise, post your command-line here.  We need to reproduce the problem before we can offer any suggestions.

**Comment by jidanni** [comment]
That's the problem, I didn't supply any font argument, because just reading the web page at that point doesn't tell me to!
(I just jumped in to the middle of that webpage from that link fragment identifier via Google or something.)
Sure, if one keeps on reading one finally sees that a font argument is needed. the first example assumes that the user knows that there are two different kinds of unicode. One that doesn't need a font argument and one that does.
Okay, that article was written back when people knew the difference between latin1 and the rest of unicode.

**Comment by urban-warrior** [comment]
We added a system policy to set the default font, the one to use if it is not specified on the command line.  For example, here we specify a Unicode Arial font that can grok multi-language character sets:

```
<policy domain="system" name="font" value="/usr/local/share/font/arial-unicode.ttf"/>
```
